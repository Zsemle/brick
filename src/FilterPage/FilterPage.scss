@import '../style/layout.scss';

.filter-page {
  @media (min-width: $bp-medium) {
    display: flex;
    max-width: 920px;
    margin: auto;
  }
}
