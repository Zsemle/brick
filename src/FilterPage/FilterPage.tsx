import React, { FunctionComponent } from 'react'
import Filter from '../Filter/Filter'
import Results from '../Results/Results'

const FilterPage:FunctionComponent = ():JSX.Element => {
  return (
    <div>
      <Filter/>
      <Results/>
    </div>
  )
}

export default FilterPage
