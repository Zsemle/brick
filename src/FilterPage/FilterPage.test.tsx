import { shallow, ShallowWrapper } from 'enzyme'
import React from 'react'
import FilterPage from './FilterPage'

describe('the filter page component', () => {
  it('renders the component', () => {
    const wrapper:ShallowWrapper = shallow(<FilterPage/>)
    expect(wrapper).toMatchSnapshot()
  })
})
