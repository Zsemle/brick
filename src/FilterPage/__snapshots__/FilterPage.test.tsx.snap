// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the filter page component renders the component 1`] = `
<div>
  <Filter />
  <Results />
</div>
`;
